app/styles/components/_large-search.scss

```diff
@@ -27,8 +27,7 @@
     }
 
     .no-results {
-      color: white;
-      margin-top: 0.5rem;
+      margin-top: 0.5rem 0;
```

Comment:
Did you mean to change this to `margin` or maybe just an accidental 0 at the end?

Reply:
@kategengler I mean to delete the white color, also I added the margin because looks better with it

Reply:
@kexposito The `margin-top` was existing, but you added ` 0` after the `0.5rem'. `margin-top` does not take two values https://developer.mozilla.org/en-US/docs/Web/CSS/margin-top so I thought you may have meant to change it to `margin`

Reply:
@kategengler You right, missed that!, will fix it

Reply:
@kategengler done!